### DISCUSSION TITLE
Can @conform-to/zod's parse function return all formData errors?

### DISCUSSION THREAD

**Discussion by setvik** [edmundhung/conform#280] [Q&A] [answered]
Prototyping with conform-to and trying to figure out how to parse form data with @conform-to/zod's parse function and get it to return all errors in the form, not just the first error encountered.

Is that possible?

For example, if I submit a form with two required fields, subject and body, and then parse the formData with @conform-to/zod's parse function as follows, it will only report the subject field as missing, whereas I want it to report that both are missing (so user can see both issues at the same time, as opposed to revealing the missing body field only after the user populates the subject field):

```
import { z } from 'zod';
import { parse } from '@conform-to/zod';
...
const parsed = parse(formData, { schema:
  z.object({
    subject: z.string({ required_error: 'subject required' }),
    body: z.string({ required_error: 'body required' }),
  })
});
```

**Comment by edmundhung** [comment, marked as answer]
Hi @setvik, there is currently a bug on zod 3.22 that mess up the validation. Please downgrade to zod 3.21 first.

Reference: #271

**Comment by edmundhung** [comment]
It should be resolved on [v0.8.1](https://github.com/edmundhung/conform/releases/tag/v0.8.1)
